### ISSUE TITLE
oAuth 1.0a nonce_used

### ISSUE THREAD

**Issue by kidager** [a85/POSTMan-Chrome-Extension#2737]
**_oAuth 1.0a_** does not regenerate the **_nonce_** value on the first request

 1. Postman Version: 4.10.2
 2. App (Chrome app or Mac app): Mac app
 3. OS details: macOS Sierra 10.12.3 (16D32)
```
$ uname -a
Darwin nowhere.local 16.4.0 Darwin Kernel Version 16.4.0: Thu Dec 22 22:53:21 PST 2016; root:xnu-3789.41.3~3/RELEASE_X86_64 x86_64
```
 4. Is the Interceptor on and enabled in the app: No
 5. Did you encounter this recently, or has this bug always been there: Always
 6. Expected behaviour: Regenerate nonce key and then send the request
 7. Console logs (http://blog.getpostman.com/2014/01/27/enabling-chrome-developer-tools-inside-postman/ for the Chrome App, View->Toggle Dev Tools for the Mac app):
```
🐶 watchdog checking connection health
```
 8. Screenshots (if applicable)

Steps to reproduce the problem:
- Create a request that uses oAuth 1.0a
- Save it into collections
- Close it
- Reopen the same request (It should not be edited)
- Send the request

**Comment by vegetableman** [comment]
@kidager This could be a conflict of interest with other users using oAuth. As, some users prefer sending the stored `nonce` when the request is loaded for the first time. cc @a85, @SamvelRaja.

**Comment by harryi3t** [comment]
@kidager This will be fixed in Postman 5.3
It will be generated by Postman if the field is left empty.
Any field that has a user input will not be overridden by Postman. This will be true for all auths
